workflow/scripts/get_genome.py

```
@@ -0,0 +1,159 @@
+# This script attempts to download genome sequence (FASTA) and
+# genome annotation (GFF / GTF) files from NCBI using the NCBI datasets
+# API, or a similar database. Also, the genome sequence is indexed using samtools.
+# Alternatively, a FASTA and GFF file can be
+# supplied by the user. Input is roughly checked for validity.
+
+from os import path
+from BCBio.GFF import GFFExaminer
+from BCBio import GFF
+from subprocess import getoutput
+
+input_database = snakemake.params["database"]
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)